python/tank/bootstrap/cached_configuration.py

```diff
@@ -182,13 +182,11 @@ def status(self):
 
         # Pass 1:
         # first check if there is any config at all
-        # probe for info.yaml manifest file
-        sg_config_file = os.path.join(
+        sg_config_folder = os.path.join(
             self._path.current_os,
-            "config",
-            constants.BUNDLE_METADATA_FILE
+            "config"
```

Comment:
We used to look for the `config/info.yml` , which was written inside the cached configuration (inside `~/Library/Caches/Shotgun/site` for example). The problem with that logic is that since the config was no more copied inside the cached configuration it was always considered as missing and the whole configuration was redeployed.

Now we're only looking for the `config` folder.

Reply:
👍